update/repository.go

```diff
@@ -150,3 +151,18 @@ func (r *GormRepository) Stop() error {
 	}
 	return nil
 }
+
+const TenantsUpdateAdvisoryLockID = 4242
+
+func lock(do func(repo Repository) error) dbsupport.LockAndDo {
```

Comment:
why don't you move this `lock` function in the new `dbsupport` package? it seems like a common function that can be reused accros multiple repositories, doesn't it?

Reply:
done 32ee892

Reply:
great, thanks!